src/PropertyInfo/Extractors/GetterExtractor.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * (c) Mihai Stancu <stancu.t.mihai@gmail.com>
```

Comment:
The copyright header must be the one of the project:

```
/*
 * (c) Kévin Dunglas <dunglas@gmail.com>
 *
 * This source file is subject to the MIT license that is bundled
 * with this source code in the file LICENSE.
 */
```

(same for all files)